snow/engine/snowman/block/batched_vm_test.go

```diff
@@ -25,7 +26,7 @@ func TestGetAncestorsDatabaseNotFound(t *testing.T) {
 		require.Equal(t, someID, id)
 		return nil, database.ErrNotFound
 	}
-	containers, err := GetAncestors(context.Background(), vm, someID, 10, 10, 1*time.Second)
+	containers, err := GetAncestors(context.Background(), logging.NewLogger("test"), vm, someID, 10, 10, 1*time.Second)
```

Comment:
hmmm we've always used the `logging.NoLog{}` for this... Probably best to stay in-line with that... but could think about just removing `logging.NoLog{}` entirely in a future PR.